src/utils/urlbuilder.js

```diff
@@ -0,0 +1,33 @@
+function getCDNBasePath(stage) {
```

Comment:
This is not really the path. We could call it 
```suggestion
function getCDNDomain(stage) {
```

Reply:
Fair enough